ocs/site_config.py

```diff
@@ -430,6 +432,32 @@ def get_config(args, agent_class=None):
 
     return (site_config, host_config, instance_config)
 
+
+def add_site_attributes(args, site, host=None):
+    """
+    Adds site and host attributes to namespace if they do not exist.
+
+    Args:
+        args:
+            namespace to add attributes to.
+        site:
+            Site config object.
+        host:
+            Host config object.
+    """
+    if args.site_hub is None:
+        args.site_hub = site.hub.data['wamp_server']
+    if args.site_http is None:
+        args.site_http = site.hub.data.get('wamp_http')
+    if args.site_realm is None:
+        args.site_realm = site.hub.data['wamp_realm']
+    if args.address_root is None:
+        args.address_root = site.hub.data['address_root']
+    if args.registry_address is None:
+        args.registry_address = site.hub.data.get('registry_address')
+    if (args.log_dir is None) and (host is not None):
+        args.log_dir = host.log_dir
+
 def reparse_args(args, agent_class=None):
```

Comment:
Perhaps mark this function, with a comment or in the docstring, as deprecated in favor of the simpler site_config.parse_args call.